paypal/standard/helpers.py

```diff
@@ -24,11 +33,12 @@ def duplicate_txn_id(ipn_obj):
     """
 
     # get latest similar transaction(s)
-    similars = (ipn_obj.__class__._default_manager
-                .filter(txn_id=ipn_obj.txn_id)
-                .exclude(id=ipn_obj.id)
-                .exclude(flag=True)
-                .order_by('-created_at')[:1])
+    similars = (
+        ipn_obj.__class__._default_manager.filter(txn_id=ipn_obj.txn_id)
+        .exclude(id=ipn_obj.id)
+        .exclude(flag=True)
+        .order_by("-created_at")[:1]
+    )
```

Comment:
For next time - having reformatting as well as code changes makes things quite difficult to read...